runelite-api/src/main/java/net/runelite/api/coords/WorldPoint.java

```diff
@@ -191,6 +195,58 @@ public static WorldPoint fromLocalInstance(Client client, LocalPoint localPoint)
 		}
 	}
 
+	/**
+	 * Gets a {@link Collection} of possible {@link WorldPoint}s given a passed {@link net.runelite.api.Scene} ID, and
+	 * coordinates.
+	 *
+	 * @param client  The client, used to determine whether it is running in an instanced Scene
+	 * @param sceneId The ID of the Scene
+	 * @param sceneX  The X point within the Scene
+	 * @param sceneY  The Y point within the Scene
+	 * @param z       The Z point within the Scene
+	 * @return A {@link Collection} of {@link WorldPoint}s which can correspond to the given
+	 *         {@link net.runelite.api.Scene} values. Should return a singleton list when run outside of an instance.
+	 */
+	public static Collection<WorldPoint> fromScene(final Client client, final int sceneId, final int sceneX, final int sceneY, final int z)
```

Comment:
Use the method added in #7788 instead. Also `sceneId` isn't a thing we have defined anywhere.